src/cdk/accordion/accordion.ts

```diff
@@ -28,4 +32,20 @@ export class CdkAccordion {
   get multi(): boolean { return this._multi; }
   set multi(multi: boolean) { this._multi = coerceBooleanProperty(multi); }
   private _multi: boolean = false;
+
+  /** Expands all enabled accordion items in an accordion where multi is enabled. */
+  expandAll(): void {
```

Comment:
These should likely be `openAll` and `closeAll` to match the naming convention used in `CdkAccordionItem`.

Reply:
Done.